### ISSUE TITLE
Long loading availability holdings when clicked View More of a location with many holdings

### ISSUE THREAD

**Issue by banukutlu** [psu-libraries/psulib_blacklight#504]
https://catalog.libraries.psu.edu/catalog/71224 - failing

https://catalog.libraries.psu.edu/catalog/189479 - big but does work, but maybe we could use this to help benchmark?

(also on the My Account side -- how do these behave on loading there? options for improving that? - would need to then be my acct ticket)

**Comment by nicktiberi** [comment]
Per accessibility testing with Michelle/Ruth/Binky, we should do the following to make it easier for screen reader users to navigate the holdings table:

- change "More Holdings" to say "Holdings x - y" and make it a h6.
- add "Holdings x-y" above each "page" of holdings
- add previous/next buttons next to the "Holdings  x-y" headings for easier navigation
